R/utilities-units.R

```diff
@@ -347,14 +348,18 @@ getUnitsEnum <- function() {
         getUnitsForDimension(dimension = dimension)
       },
       error = function(cond) {
-        warning(messages$errorLoadingUnitsForDimension(dimension, cond))
+        errors <- c(errors, dimension)
         # making sure that in this case, the user sees that something went wrong
         return(c("Unavailable"))
       }
     )
     return(enum(replace(x, x == "", "Unitless")))
   })
 
+  if (length(errors) > 0L) {
+    message(messages$errorLoadingUnitsForDimensionoadingUnitsForDimension(errors))
```

Comment:
```suggestion
    message(messages$errorLoadingUnitsForDimension(errors))
```

Reply:
sure this is copy paste BS. But why isn't the code working?